src/generator/css/base.less

```diff
@@ -659,6 +863,237 @@ body .content .hero-banner {
     @apply bg-opacity-50;
 }
 
+/** Grid */
+.what-we-do {
+  @apply bg-[var(--color-accent-dark)] py-24 sm:py-52 px-0 sm:px-20 -mt-32 z-10 mb-40;
+  clip-path: polygon(100% 4.5%, 0% 0% ,0% 100%, 100% 93.5%);
+  position: relative;
+
+  h2 {
+    font-family: var(--font-h1);
+    font-weight: 700;
+    font-size: 28px;
+    @apply pl-5 sm:pl-0 mb-16 sm:mb-28 -ml-8;
+    border: unset;
+    span {
+      @apply text-[var(--color-accent)];
+    }
+
+    @media (max-width: 422px) {
+        font-size: 28px;
+      }
+
+    @media (min-width:400px) and (max-width: 640px) {
+        margin-bottom: 8rem;
+        font-size: 28px;
+    }
+  }
+  
+
+  .grid {
+    grid-template-columns: repeat(12, 1fr);
+    @apply gap-y-16;
+    
+    @media (min-width: 820px) {
+        grid-template-rows: repeat(3, 1fr); 
+    }
+
+    @media (max-width: 820px) {
+        grid-template-rows: repeat(3, 1fr);
+        @apply gap-y-8;
+    }
+
+    @media (min-width: 430px) and (max-width: 640px) {
+        @apply gap-y-20;
+    }
+    @media (min-width: 520px) and (max-width: 820px) {
+        @apply gap-y-28;
+    }
+  }
+
+  .grid > :nth-child(1),
+  .grid > :nth-child(2) {
+    grid-row: 1;
+  }
+
+  .grid > :nth-child(3),
+  .grid > :nth-child(4) {
+    grid-row: 2;
+  }
+
+  .grid > :nth-child(5),
+  .grid > :nth-child(6) {
+    grid-row: 3;
+  }
+
+  .grid > :nth-child(2),
+  .grid > :nth-child(3),
+  .grid > :nth-child(6),
+  .grid > :nth-child(7) {
+    @apply z-10 my-8 px-6 sm:px-12 pt-16 pb-20;
+    background-color: var(--color-white-primary);
+    align-self: center;
+
+    p {
+      @apply text-sm md:text-xl tracking-normal;
+      line-height: 1.7rem;
+      color: var(--color-grey-primary);
+      white-space: unset;
+      font-weight: 400;
+
+      strong {
+        @apply tracking-widest text-base md:text-4xl mb-4 sm:mb-9 block;
+        font-family: var(--font-h1);
+        font-weight: 700;
+      }
+
+      @media (max-width: 700px) {
+       padding-top: 1rem;
+       padding-bottom: 0;
+      }
+    }
+  }
```

Comment:
When it comes to styling work in general: please keep the media breakpoints you use consistent. This makes debugging and maintaining you styles easier and more consistent. Having multiple media queries with differing pixel sizes, while it can produce better control over particular section of the page, it slows down future styling work significantly because you have to test ever resolution to find custom media breakpoints. Rather than testing known ones.